tests/testthat/helper-html.R

```diff
@@ -3,7 +3,7 @@ show_raw_html <- function(x) {
 }
 
 expect_snapshot_html <- function(x, .envir = parent.frame()) {
-  x_str <- deparse(substitute(x))
+  x_str <- deparse1(substitute(x), collapse = " ")
```

Comment:
Good catch on `deparse1()`! The default `collapse` argument is `" "` so we can drop it here

```suggestion
  x_str <- deparse1(substitute(x))
```